kratos/containers/fix_data_value_container.h

```diff
@@ -282,32 +282,23 @@ class FixDataValueContainer
 
     template<class TDataType> TDataType& GetValue(const Variable<TDataType>& rThisVariable)
     {
-        if(!mpVariablesList->Has(rThisVariable))
-            KRATOS_ERROR << "variable " << rThisVariable << " not found in the fixed_data_value_container";
-//             mpVariablesList->Add(rThisVariable);
-
-//  	  KRATOS_WATCH(rThisVariable)
+        KRATOS_ERROR_IF(rThisVariable.Key()==0) << "Trying to access unregistered (key = 0) variable " << rThisVariable.Name() << " in a call to FixDataValueContainer::GetValue." << std::endl;
```

Comment:
You are right and actually I have commented it to @RiccardoRossi  but it would be a change of behavior and we postponed it to a separate PR after release